.gitlab-ci.yml

```diff
@@ -4,8 +4,7 @@ stages:
   - publish
   - optional
 
-image:                             parity/rust:gitlab-ci
-
+image:                             parity/rust-parity-ethereum-build:xenial
```

Comment:
what's the logic behind changing this now?

Reply:
this docker image has been tested,
it's the preparation for using cache in docker volumes

Reply:
new CI images